data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -776,9 +758,11 @@ def build_academy_data(
         axis=1,
     )
 
+    # TODO: remove; duplicate of `Overall Phase`, above?
```

Comment:
Agree - assuming `SchoolPhaseType` is unused. It's not written to the db.

Reply:
It's used for maintained-schools (but so is `Overall Phase`) and used as part of the comparator-set processing.

In both cases, it'd probably be easy to remove but for the sake of keeping the changes to a minimum, I vote for picking up later.

Reply:
Agree